### ISSUE TITLE
How to have access to the file system?

### ISSUE THREAD

**Issue by abbestrov** [naviapps/create-nw-react-app#42]
In case I want to use an SQLite database with my project, how can I get access to the file system. in other words how to set up SQLite support in a project created with <create-nw-react-app>?
And there is another thing, after creating a project with <create-nw-react-app> I get the usual file structure given by <create-react-app>, the problem here how can I make changes to the window like title, width, position, and other properties?

**Comment by matteoturini** [comment]
```const fs = require('fs')``` if I remember correctly.

**Comment by acroyear** [comment]
Yeah - the thing think about is that everything available to a node.js "server" app is available in nw.js.  Simply call the file system and path methods directly.

I would recommend using the promises versions, in order to not block the UI while the file system stuff is happening. nw.js has full access to async and await which makes the code a bit more readable, too.
